_pages/relays/nostrRSRelay.md

```diff
@@ -0,0 +1,17 @@
+---
+title: nostr-rs-relay
+permalink: /nostrRSRelay/
+web: https://sr.ht/~gheartsfield/nostr-rs-relay/
+github: scsibug/nostr-rs-relay
+tags:
+- relay
+progLang:
+- Python3 
```

Comment:
Python??

```suggestion
- Rust 
```

Reply:
Hmmm, was confused with the other one also named nostr relay. 

![image](https://user-images.githubusercontent.com/44260360/226079381-3bc7b1c6-67fb-40ca-90b9-04d7a5f919c4.png)